cdc/sorter/leveldb/table_sorter.go

```diff
@@ -15,307 +15,90 @@ package leveldb
 
 import (
 	"context"
-	"math"
-	"sync/atomic"
 	"time"
 
 	"github.com/pingcap/errors"
 	"github.com/pingcap/log"
 	"github.com/pingcap/tiflow/cdc/model"
-	"github.com/pingcap/tiflow/cdc/sorter"
 	"github.com/pingcap/tiflow/cdc/sorter/encoding"
 	"github.com/pingcap/tiflow/cdc/sorter/leveldb/message"
 	"github.com/pingcap/tiflow/pkg/actor"
 	actormsg "github.com/pingcap/tiflow/pkg/actor/message"
-	"github.com/pingcap/tiflow/pkg/config"
 	"github.com/pingcap/tiflow/pkg/db"
-	"github.com/pingcap/tiflow/pkg/util"
 	"github.com/prometheus/client_golang/prometheus"
 	"go.uber.org/zap"
 )
 
-const (
-	// Capacity of db sorter input and output channels.
-	sorterInputCap, sorterOutputCap = 64, 64
-	// Max size of received event batch.
-	batchReceiveEventSize = 32
-)
-
-var levelDBSorterIDAlloc uint32 = 0
-
-func allocID() uint32 {
-	return atomic.AddUint32(&levelDBSorterIDAlloc, 1)
-}
-
-type common struct {
-	dbActorID actor.ID
-	dbRouter  *actor.Router
-
-	uid     uint32
-	tableID uint64
-	serde   *encoding.MsgPackGenSerde
-	errCh   chan error
-}
-
-// reportError notifies Sorter to return an error and close.
-func (c *common) reportError(msg string, err error) {
-	if errors.Cause(err) != context.Canceled {
-		log.L().WithOptions(zap.AddCallerSkip(1)).
-			Warn(msg, zap.Uint64("tableID", c.tableID), zap.Error(err))
-	}
-	select {
-	case c.errCh <- err:
-	default:
-		// It means there is an error already.
-	}
-}
-
-// Sorter accepts out-of-order raw kv entries and output sorted entries
-type Sorter struct {
+// reader accepts out-of-order raw kv entries and output sorted entries
+type reader struct {
```

Comment:
there is a writer.go,  maybe rename the file to reader.go is better

Reply:
I'm going to do in the next PR. By doing so, the diff is huge in this PR, hard to review.